### DISCUSSION TITLE
Are Migrations still "experimental"?  Docs seem to conflict

### DISCUSSION THREAD

**Discussion by lukewlms** [prisma/prisma#7542] [Q&A] [answered]
The "[Is Prisma 2 Ready](https://www.notion.so/Is-Prisma-2-Ready-8b3fba3eaf5b4bf3ab7102fd94f56148)" page labels Migrations as "Experimental" still.

But the [Migrate docs page](https://www.prisma.io/migrate) doesn't have any hint that it is experimental.

I'd say one of those should be updated to provide additional clarity - can a Prisma member confirm the status of Migrations?  We're starting a major new production project and it makes a significant difference in our initial setup.

Thanks!

**Comment by ryands17** [comment, marked as answer]
@lukewlms 👋 

Migrate is no more experimental and completely stable right now so you can go ahead and use it as mentioned in the docs. Thanks for pointing this out as we most likely forgot to update this. I have removed the experimental part as it isn't needed anymore :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
